Sources/Swift/Persistence/SentryScopePersistentStore+Tags.swift

```diff
@@ -0,0 +1,49 @@
+@_implementationOnly import _SentryPrivate
+
+extension SentryScopePersistentStore {
+    func encode(tags: [String: String]) -> Data? {
+        // We need to check if the Tags is a valid JSON object before encoding it.
+        // Otherwise it will throw an unhandled `NSInvalidArgumentException` exception.
+        // The error handler is required due but seems not to be executed.
+        guard let sanitizedTags = sentry_sanitize(tags) else {
```

Comment:
`m`: Do we have a test case with an invalid tag value which requires the sanitization?